CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-<!-- ### Added -->
+### Added
+
+- [Examples] Added functional tests for all examples using Cypress. [#227](https://github.com/Accenture/reactive-interaction-gateway/issues/227)
```

Comment:
I must admit I'm not a huge fan of those `[]` categories. The Changelog target _users_ and they pretty much have no clue how we organize our source code (and rightfully so). Also, even for us developers, `[Examples]` doesn't add much value since I need to read the rest of the line anyway. In this case, the sentence on its own already tells me that it's about the examples; in other cases, one might set the scene with words rather than with text. For example, I think "When forwarding outbound HTTP responses, RIG makes sure the HTTP version of the response matches what the clients expects." better than the more technical "[Proxy] HTTP procotol version-conversion for outbound responses.". Does that make sense? :)

Reply:
Ye, me neither ... I was always thinking about `[]` as a way how to filter all changes for particular module/sub-app/... but we are not following any pattern/style for that really ... what do you propose? should we get rid of them?

Reply:
I think we either have to use very few (consistently) and define them clearly, or we should omit them altogether. I lean towards the latter.